dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/domain/scope/RumViewScope.kt

```diff
@@ -1005,6 +1007,10 @@ internal open class RumViewScope(
             is Activity -> key
             is Fragment -> key.activity
             is android.app.Fragment -> key.activity
+            is NavigationViewTrackingStrategy.NavigationKey -> {
+                navControllerActivityField?.isAccessible = true
```

Comment:
maybe we can add some log entry when `navControllerActivityField` is not found (because of the private API changes, for example)? this will simplify the debugging if things don't work.

Reply:
Yes indeed, I'll add one